src/ParseFile.js

```diff
@@ -271,6 +271,25 @@ class ParseFile {
     this._requestTask = null;
   }
 
+  /**
+   * Deletes the file from the Parse cloud.
+   * @param {Object} options
```

Comment:
You can remove the options as this is masterKey only. Also document that this is master key only or to be used in cloud code.

https://github.com/parse-community/parse-server/blob/master/src/Routers/FilesRouter.js#L32